GVRf/Framework/src/org/gearvrf/GVRRenderTexture.java

```diff
@@ -55,6 +55,20 @@ public GVRRenderTexture(GVRContext gvrContext, int width, int height,
     GVRRenderTexture(GVRContext gvrContext, long ptr) {
         super(gvrContext, ptr);
     }
+    
+    /**
+     * Return the width of GVRRenderTexture (FBO)
+     */
+    public int getWidth() {
+    	return NativeRenderTexture.getWidth(getPtr());
+    }
+
```

Comment:
Do we really need these methods (and the new JNI code)? Could you just save the height and width in the constructor? (It doesn't look like these textures are being created by the native code.)